src/main/resources/schema.sql

```diff
@@ -1 +1,14 @@
-USE alkemy;
\ No newline at end of file
+USE alkemy;
+CREATE TABLE members (
```

Comment:
agreguemos IF NOT EXISTS antes del nombre de la tabla, asi solo se crea la primera vez